### DISCUSSION TITLE
Dynamically changing FPS

### DISCUSSION THREAD

**Discussion by hamsando** [blakeblackshear/frigate#426] [Ask A Question] [answered]
The Wyze v2 cameras have 3 modes of operation.
Day - 15 fps (color)
Night - 10 fps (b/w - not IR lit)
Auto - switches between the above two, depending on available light.

Since frigate/ffmpeg is looking for and uses the fps, is it problematic to have the fps dynamically switch twice/day?

**Comment by blakeblackshear** [comment, marked as answer]
It should be fine. Personally, I think 5fps is ideal. Anything more and it seems to be overkill in my experience.

**Reply by hamsando** [reply]
I understand the 5fps, and I've seen you mention it repeatedly in the HA forum thread, unfortunately there is no way to set it on the Wyze cam. It is a fixed rate depending on what mode you operate in.

I've been leaving it running on Night mode 24/7 and could do a take_frame: 2 to cut that to 5fps. 

I was just trying to get an understanding if it would be problematic to have the fps change dynamically (which you think it won't).
Thanks!

**Reply by blakeblackshear** [reply]
If you set fps to 5 in the camera config, ffmpeg will reduce it for you.

**Comment by bm55** [comment]
> The Wyze v2 cameras have 3 modes of operation.
> Day - 15 fps (color)
> Night - 10 fps (b/w - not IR lit)
> Auto - switches between the above two, depending on available light.
> 
> Since frigate/ffmpeg is looking for and uses the fps, is it problematic to have the fps dynamically switch twice/day?

Hi @hamsando,
Does Wyze cam has option for RTMP with their RTSP frimware? When I tried last year with RTSP stream but it wasn't providing consistent fps which caused a lot of issue with frigate.

**Reply by paularmstrong** [reply]
I have a couple Wyze cameras running with the RTSP firmware and the 0.8.0 releases are working great

**Reply by hamsando** [reply]
My understanding is that there are 2 possible fw's you could use on the Wyze V2 for rtsp.
Dafang or the the Wyze beta(?) RTSP fw. I'm using the Wyze one (and no, there is not a RTMP option)

For Frigate, I'm using the 0.7.3 release on a Pi4. Originally I was running without a Coral and I had no end of problems (I think the pi was too busy and so the connection would drop regularly). However once I threw the Coral in, it seems to be pretty solid.
